packages/common-types/src/earn.ts

```diff
@@ -29,6 +29,14 @@ type ApiGuardianFee = {
   percentage: number;
 };
 
+export type ApiEarnToken = {
```

Comment:
We do have this type somewhere right? or is it exported from the sdk directly?

Reply:
We have `DCAPositionToken`:
```
export type DCAPositionToken = {
    address: TokenAddress;
    symbol: string;
    name: string;
    decimals: number;
    price?: number;
    variant: TokenVariant;
};
I supposed we would need a new type for earn, so I created this new `ApiEarnToken`

Reply:
Mmm, lets check if we can import it from the sdk directly, if not merge it as is